src/main.ts

```diff
@@ -1,5 +1,5 @@
 // For more information, see https://crawlee.dev/
-import { PlaywrightCrawler } from "crawlee";
```

Comment:
Had to revert changes, it was capturing 'www.site.com/img.png" from links on pages and taking up an unnecessarily large amount of space in the output.json. I'd suggest keeping this feature, but adding an option in config.ts to enable or disable it, or enable with a url blacklist option.

Reply:
okkk, nice gotcha, let me see what i can do about the image issue

Reply:
following you suggestion, I added a extension blacklist in the config.ts, if there is a match in the route with the blocked extension, then the request its going to be aborted.